### DISCUSSION TITLE
How do we interpret the info in this mod?

### DISCUSSION THREAD

**Discussion by joeynuggetz** [Fetty42/FS22_AnimalFoodOverview#3] [General]
Not sure I understand the percentages given under the "All Food Types" section. What do the percentages mean exactly? Can you guys provide some information as I want to use this tool but its not clear what certain percentages mean.

**Comment by Fetty42** [comment]
Hi, 
"Prod. Weight" stands for how much the feed increases productivity. 1.00 is the maximum.
"Eat Weight" indicates how much of this feed the animals consume. Usually identical to "Prod. Weight".

In the case of parallel consumption, the values for "Prod. Weight" are added together.
As far as I know, you need at least 75% "Prod. Weight" for the productivity to be displayed.
